homeassistant/components/overseerr/sensor.py

```diff
@@ -24,51 +23,87 @@
 class OverseerrSensorEntityDescription(SensorEntityDescription):
     """Describes Overseerr config sensor entity."""
 
-    value_fn: Callable[[RequestCount], int]
+    value_fn: Callable[[OverseerrData], int]
 
 
 SENSORS: tuple[OverseerrSensorEntityDescription, ...] = (
     OverseerrSensorEntityDescription(
         key="total_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.total,
+        value_fn=lambda data: data.requests.total,
     ),
     OverseerrSensorEntityDescription(
         key="movie_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.movie,
+        value_fn=lambda data: data.requests.movie,
     ),
     OverseerrSensorEntityDescription(
         key="tv_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.tv,
+        value_fn=lambda data: data.requests.tv,
     ),
     OverseerrSensorEntityDescription(
         key="pending_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.pending,
+        value_fn=lambda data: data.requests.pending,
     ),
     OverseerrSensorEntityDescription(
         key="declined_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.declined,
+        value_fn=lambda data: data.requests.declined,
     ),
     OverseerrSensorEntityDescription(
         key="processing_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.processing,
+        value_fn=lambda data: data.requests.processing,
     ),
     OverseerrSensorEntityDescription(
         key="available_requests",
         native_unit_of_measurement=REQUESTS,
         state_class=SensorStateClass.TOTAL,
-        value_fn=lambda count: count.available,
+        value_fn=lambda data: data.requests.available,
+    ),
+    OverseerrSensorEntityDescription(
+        key="total_issues",
+        native_unit_of_measurement=ISSUES,
```

Comment:
Oh we can move the unit of measurement to the translations and then we can translate it :)